frontend/src/pages/AuthenticatedHomePage.tsx

```diff
@@ -10,9 +10,12 @@ import {
 } from '@mui/material'
 
 export function AuthenticatedHomePage() {
-  const connectBankClicked = () => {
-    alert("Let's connect your bank!")
-    // TODO: Open the Link widget here.
+
+  // Retrieve Link Token from Backend and log it in the browser
+  const connectBankClicked = async() => {
+    const response = await fetch("http://localhost:3000/api/banking/create_link_token",{method:"Get"});
+    const data = await response.json();
+    console.log(data);
```

Comment:
Nit: we usually don't want to check in console.logs in production code. please remove.

Reply:
Now I know